examples/camera_params.py

```diff
@@ -24,20 +24,22 @@ async def run():
     drone = System()
     await drone.connect(system_address="udpin://0.0.0.0:14540")
 
-    asyncio.ensure_future(observe_current_settings(drone))
-    asyncio.ensure_future(observe_camera_mode(drone))
-    asyncio.ensure_future(observe_possible_setting_options(drone))
+    _tasks = [
+        asyncio.create_task(observe_current_settings(drone)),
+        asyncio.create_task(observe_camera_mode(drone)),
+        asyncio.create_task(observe_possible_setting_options(drone)),
+    ]
```

Comment:
This syntax would delete tasks as they are completed, as discussed in `ruff rule RUF006`.
```suggestion
    tasks = {
        asyncio.create_task(observe_current_settings(drone)),
        asyncio.create_task(observe_camera_mode(drone)),
        asyncio.create_task(observe_possible_setting_options(drone)),
    }
    for task in tasks:
        task.add_done_callback(tasks.discard)
```